doc/sphinx/proofs/automatic-tactics/auto.rst

```diff
@@ -329,7 +329,9 @@ and `Constants`, while implicitly created databases have the `Opaque` setting.
 
 .. cmd:: Create HintDb @ident {? discriminated }
 
-   Creates a new hint database named :n:`@ident`. The database is
+   If a hint database named :n:`@name` already exists, do nothing.
+
+   Otherwise creates a new hint database named :n:`@ident`. The database is
```

Comment:
```suggestion
   If there is no hint database named :n:`@name`, creates a new hint database
   with that name.  Otherwise, does nothing.  The database is
```

Better to mention the common case before the exceptional case.